tests/declare/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`declare.php 1`] = `
+<?php
+declare(ticks=1):
+$test = 1;
+enddeclare;
+declare(ticks=1);
+$test = 1;
+declare(ticks=1) {
+  $test = 1;
+}
+declare(encoding='ISO-8859-1');
+$test = 1;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+declare(ticks=1):
```

Comment:
In javascript we put spaces around =, is it idiomatic not to in php?

Reply:
typically yes - this is a bit of an edge case, its a system level directive http://php.net/manual/en/control-structures.declare.php